test/aliceO2-namespace-naming.cpp

```diff
@@ -0,0 +1,152 @@
+// RUN: %check_clang_tidy %s aliceO2-namespace-naming %t
+
```

Comment:
can you add here:
a) a double nested namespace example
b) a `using namespace Foo` example
c) a simple typedef aka `using MyType = Foo::Bar::ClassA`